docs/commands.md

```diff
@@ -50,7 +50,10 @@ function setSecret(secret: string): void {}
 
 Now, future logs containing BAR will be masked. E.g. running `echo "Hello FOO BAR World"` will now print `Hello FOO **** World`.
 
-**WARNING** The add-mask and setSecret  commands only support single line secrets. To register a multiline secrets you must register each line individually otherwise it will not be masked.
+**WARNING** The add-mask command only supports single line secrets. To register
+a multiline secret you must register each line individually otherwise it will
+not be masked, if you use `@actions/core >= 1.11.0` `setSecret` will do this
+for you.
```

Comment:
```suggestion
**WARNING** The add-mask command only supports single-line secrets. To register
a multi-line secret, the recommended practice is to register each line individually. Otherwise, it will
not be masked. `@actions/core >= 1.11.0` `setSecret` will perform this automatically.
```